documentation/docs/pages/Parameter-choices.md

```diff
@@ -0,0 +1,251 @@
+
+Another parameter we adjust is `HFREEZE = 10.0`. This means the model computes a "melt potential" over a `10m` layer for sea-ice melt/freeze processes. If `HFREEZE >0`, the ocean will calculate how much heat is available in the top 10 meters to melt ice. 
+
+### Sea surface forcing
+The current configuration is forced with repeating atmospheric conditions and includes weak surface restoring for salinity. We use a repeat-year forcing (RYF) strategy with the JRA55-do v1.6 dataset. This means the model experiences one year of normal seasonal cycle atmospheric fluxes (wind, heat, precipitation, etc) that is repeated every model year. The advantages are avoiding interannual variability while maintaining a realistic seasonal cycle, helping to equilibrate the ocean state. More details can be found in [Forcing data models](pages/forcing-data-models/).
+
+### Surface salinity restoring
+We use `RESTORE_SALINITY = True` to have sea surface salinity (`SSS`) is weakly restored to a reference field (`SALT_RESTORE_FILE = "salt_sfc_restore.nc"`), and a piston velocity (`FLUXCONST = 0.11`m/day) is given. The restoring is implemented as a virtual salt flux ( `SRESTORE_AS_SFLUX = True`) to nudge `SSS`. This approach conserves salt overall (balanced globally by subtracting the mean flux, because we set `ADJUST_NET_SRESTORE_TO_ZERO = True` to avoid altering global salinity). `MAX_DELTA_SRESTORE = 999` hence currently there’s effectively no cap on the `SSS` anomaly used. More discussions and decisions can be found at [issues/350](https://github.com/ACCESS-NRI/access-om3-configs/issues/350), [issues/325](https://github.com/ACCESS-NRI/access-om3-configs/issues/325), [issues/257](https://github.com/ACCESS-NRI/access-om3-configs/issues/257).
+
+### Diagnostics and age tracer
+The current configuration intrigers some passive tracers and diagnostics for analysis. For example, we enable `USE_IDEAL_AGE_TRACER = True`, which measures the time since water left the surface. This tracer is incremented by +1 year per year everywhere and reset to 0 at the surface when water is in contact with the atmosphere. It doesn’t affect dynamics but is a diagnostic to understand water mass ventilation and residence times. The cost is minimal, so it was turned on as a useful diagnostic. In the ideal_age module, we also set `DO_IDEAL_AGE = True`.
```

Comment:
```suggestion
The current configuration intrigers some passive tracers and diagnostics for analysis. For example, we enable `USE_IDEAL_AGE_TRACER = True`, which measures the time since water left the surface. This tracer ages at a rate of 1/year once it is isolated from the surface (`DO_IDEAL_AGE = True`). It doesn’t affect dynamics but is a diagnostic to understand water mass ventilation and residence times.
```